doc/sphinx-guides/source/installation/r-rapache-tworavens.rst

```diff
@@ -80,7 +82,7 @@ R is used both by the Dataverse application, directly, and the TwoRavens compani
 
 Two distinct interfaces are used to access R: Dataverse uses Rserve; and TwoRavens sends jobs to R running under rApache using Rook interface. 
 
-We provide a shell script (``conf/R/r-setup.sh`` in the Dataverse source tree; you will need the other 3 files in that directory as well - `https://github.com/IQSS/dataverse/conf/R/ <https://github.com/IQSS/dataverseconf/R/>`__) that will attempt to install the required 3rd party packages; it will also configure Rserve and rserve user. rApache configuration will be addressed in its own section.
+We provide a shell script (``conf/R/r-setup.sh`` in the Dataverse source tree; you will need the other 3 files in that directory as well - `https://github.com/IQSS/dataverse/tree/develop/conf/R <https://github.com/IQSS/dataverse/tree/develop/conf/R>`__) that will attempt to install the required 3rd party packages; it will also configure Rserve and rserve user. rApache configuration will be addressed in its own section.
```

Comment:
Don, thanks for fixing the typo in the URL. 
But I'm not sure instructing the user to download these files from the *develop* branch is the right thing either; unless they are a developer, setting up a dev. environment. If they are installing a specific release of the dataverse, it should be something like, for example, https://github.com/IQSS/dataverse/tree/v4.4/conf/R ...

Thanks!